pkgs/tools/virtualization/linode-cli/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, buildPerlPackage, perlPackages }:
+
+buildPerlPackage rec {
+  name = "linode-cli-${version}";
+  version = "1.4.7";
+
+  src = fetchFromGitHub {
+    owner = "linode";
+    repo = "cli";
+    rev = "v${version}";
+    sha256 = "1wiz067wgxi4z4rz4n9p7dlvx5z4hkl2nxpfvhikl6dri4m2nkkp";
+  };
+
+  # src = fetchgit {
+  #   url = "git://github.com/linode/cli";
+  #   rev = "8d475f3ff161b64dbe1aff167897974f92475a3c"; 
+  #   sha256 = "1c24c7n3fq9c3yibw1i4zlk3hdcjaqcd5cq4kvvw4hfg2l3w6a21";
+  # };
+
+  # Has no docs output
+  outputs = [ "out" ];
```

Comment:
The default outputs appears to contain "devdoc" at well which caused this build to fail.